src/views/TheHome.vue

```diff
@@ -33,22 +39,27 @@
 export default {
   data() {
     return {
+      isLogin : true,
       username: '',
       password: ''
     };
   },
 
+  mounted() {
+  },
+
   computed: {
-    isLogin(){
-      return this.$route.name == 'TheHome'
-    }
+   
   },
 
   methods: {
     submitForm() {
       console.log(`Username: ${this.username} Password: ${this.password}`);
       // Aqui você pode enviar as informações do formulário para o seu back-end
-    }
+    },
+    toogleIsLogin(status){
```

Comment:
`toggle`

Reply:
`Boas práticas:` Se um método é chamado `toggle` significa que ele vai inverter o valor da booleana. Por esse motivo você não precisa passar como parâmetro nenhum valor a ser colocado na variável, porque a única função desse método é negar ela.

Por exemplo:
Para `isModalOpen`
```
toggleModal () {
  this.isModalOpen = !this.isModalOpen
}
```

Se o valor for `true`, se tornará `false` e vice-versa.

Se você tiver que passar um valor para ser colocado na variável então estamos falando de métodos específicos para `open` e `close` ou que contém alguma outra lógica que irá utilizar o parâmetro.

Reply:
ok